test/Docker.Testify/DockerSetup.cs

```diff
@@ -20,7 +20,7 @@ public abstract class DockerSetup : IDisposable
         public abstract int InternalPort { get; }
 
         public virtual string ImageTag => "latest";
-        public virtual TimeSpan TimeOut => TimeSpan.FromSeconds(30);
+        public virtual TimeSpan TimeOut => TimeSpan.FromSeconds(60);
```

Comment:
Could we just override this for elasticsearch, instead of everyone?

Reply:
Totally can ! Will do it later today :)